src/features/twitter/controllers/twitter-controller.ts

```diff
@@ -3,9 +3,11 @@ import jwt from 'jsonwebtoken';
 import type { TokenPayload } from '@/shared/infra/jwt/jwt';
 import type { Controller } from '@/shared/protocols/controller';
 import type { AsyncRequestHandler } from '@/shared/protocols/handlers';
+import { HttpStatusCode } from '@/shared/protocols/http-client';
 
 import { generateAuthURL } from '../helpers/generate-auth-url';
 import type { AuthorizeTwitterService } from '../services/authorize-twitter-service';
+import type { PostTwitterService } from '../services/post-twitter-service';
```

Comment:
Acho que é uma boa refatorar os imports pra manter o padrão com alias @:
```suggestion
import { generateAuthURL } from '@/features/twitter/helpers/generate-auth-url';
import type { AuthorizeTwitterService } from '@/features/twitter/services/authorize-twitter-service';
import type { PostTwitterService } from '@/features/twitter/services/post-twitter-service';
```